packages/design-system/ag-grid-theme/scripts/build.js

```diff
@@ -24,17 +22,23 @@ function buildStyles(entry) {
         // `node_modules`.
         findFileUrl(url) {
           if (!url.startsWith("~")) return null;
+
+          const nonPrefixedUrl = url.substring(1);
+          const resolvedFile = require.resolve(
+            nonPrefixedUrl.substring(0, nonPrefixedUrl.indexOf("/"))
```

Comment:
cc @ararus @origami-z 

I edited the build script slightly to be less sensitive to position of node_modules.